ui/apps/platform/src/Containers/Clusters/Components/Scanner/ScannerStatus.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+
+import { DetailedTooltipOverlay, Tooltip } from '@stackrox/ui-components';
+import { ClusterHealthStatus } from '../../clusterTypes';
+import {
+    delayedScannerStatusStyle,
+    healthStatusStyles,
+    isDelayedSensorHealthStatus,
+} from '../../cluster.helpers';
+import HealthLabelWithDelayed from '../HealthLabelWithDelayed';
+import { getDistanceStrictAsPhrase } from '../../../../utils/dateUtils';
```

Comment:
There are aliases for top-level directories, so you can avoid the deeply nested ancestor path for `dateUtils` by referencing it directly in the `utils/` path.

(It also has to move up in the list, because of lint rule that "absolute" paths come first.)

```suggestion
import { DetailedTooltipOverlay, Tooltip } from '@stackrox/ui-components';
import { getDistanceStrictAsPhrase } from 'utils/dateUtils';

import { ClusterHealthStatus } from '../../clusterTypes';
import {
    delayedScannerStatusStyle,
    healthStatusStyles,
    isDelayedSensorHealthStatus,
} from '../../cluster.helpers';
import HealthLabelWithDelayed from '../HealthLabelWithDelayed';
```

Reply:
Thanks for hint, fixed.